packages/components/src/GridLayout/Grid.component.js

```diff
@@ -40,7 +43,9 @@ function Grid({
 }) {
 	return (
 		<ResponsiveGridLayout
-			className="layout"
+			className={classNames(css.layout, 'layout', {
```

Comment:
Just saying
- css.layout seems to do nothing as the class is not defined in the scss file
- You can use getTheme from the theme.js file to get rid of the double declaration (and the import of classnames here)

Reply:
> css.layout seems to do nothing as the class is not defined in the scss file

Yes, it does. 
https://github.com/Talend/ui/pull/2613/files#diff-78b74fcfd7d29cbea99b03ac9503e68fR1

Reply:
Perhaps a demo could be added @ncomont 😉